libavfilter/qsvvpp.c

```diff
@@ -857,6 +850,24 @@ int ff_qsvvpp_init(AVFilterContext *avctx, QSVVPPParam *param)
     return ret;
 }
 
+static int qsvvpp_init_vpp_session(AVFilterContext *avctx, QSVVPPContext *s)
+{
+    int ret;
+
+    if (s->vpp_initted)
+        return 0;
+
+    ret = MFXVideoVPP_Init(s->session, &s->vpp_param);
```

Comment:
It may exit early in AVFilterPad::config_props callback if the sequence parameters are wrong so  I don't move  MFXVideoVPP_Query here.  I may try another MFXVideoVPP_Query call here.

Reply:
@wenbinc-Bin  I added MFXVideoVPP_Query here in the new version.

Reply:
Thanks